### ISSUE TITLE
Clash with other libraries

### ISSUE THREAD

**Issue by Bessonov** [cartant/eslint-plugin-rxjs#75]
It seems like the rules aren't check the source of functions but names only:

```typescript
import {
	equals,
	takeWhile,
	toPairs,
} from 'remeda'

[...]

return takeWhile(
		sizesAsArray,
		([_, width]) => w.innerWidth >= width,
	)
```
Triggers:
```
38:3   error  Ignoring the value within takeWhile is forbidden  rxjs/no-ignored-takewhile-value
```

**Comment by cartant** [comment]
Should be fixed in 3.3.4.

**Comment by Bessonov** [comment]
@cartant wow, it's fantastic, thank you very much for quick resolution! I can confirm that it works for me.
